app/src/main/java/agency/five/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,69 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.ui.theme.Spacing
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.Alignment
+
+@Composable
+fun UserScoreProgressBar(
+    percentage: Float,
+    value: Float,
+    modifier: Modifier = Modifier,
+    color: Color,
+    strokeWidth: Float,
+) {
+    val localSpacing = Spacing()
+    Box(
+        modifier = modifier,
+        contentAlignment = Alignment.Center
+    ){
+        Canvas(
+            modifier = modifier
+                .fillMaxSize()
+                .padding(localSpacing.extraSmall)
+        ) {
+            drawArc(
+                color = color,
+                startAngle = -90f,
+                sweepAngle = 360f * percentage,
```

Comment:
Extract all magic (hard-coded numbers) to top-level constants, example: `private const val STARTING_POSITION_OF_CIRCLE = -90f`